backEnd/main.py

```diff
@@ -1,12 +1,23 @@
-from fastapi import FastAPI
+from fastapi import FastAPI, Depends, status, HTTPException
+from fastapi.security import OAuth2AuthorizationCodeBearer, OAuth2, OAuth2PasswordRequestForm
 from pydantic import BaseModel
-
-#from routes import auth, course, user
+from routes import auth
+from fastapi.middleware.cors import CORSMiddleware
 
 app = FastAPI()
 
-#app.include_router(auth.router, prefix="/api/v1/auth", tags=["Authentication"])
-# Uncomment if you have course and user implemented
+
+app.add_middleware(
+    CORSMiddleware,
+    # Allows all origins, but be more restrictive in production
```

Comment:
Can we configure the project to all all origins in dev environment only. so we don't have to Omit this manually in production.
We can do this in ``` .Net ``` and i think in ``` Node js ``` too, what about here in FastAPI?